plugins/polio/js/src/constants/types.ts

```diff
@@ -133,3 +134,64 @@ export type BudgetEvent = {
     internal: boolean;
     amount?: number;
 };
+
+export type CampaignLogDetail = {
+    id: number;
+    content_type: string;
+    object_id: string;
+    source: string;
+    user: Record<string, any>;
+    created_at: string;
+};
+
+export type CampaignLogsDetail = Pagination & {
+    list: CampaignLogDetail[];
+};
+
+export type CampaignLogData = {
+    user: Profile;
+    new_value: Record<string, any>;
+};
+
+export type CampaignFieldType =
+    | 'string'
```

Comment:
@quang-le is there a better way for this ?

Reply:
Not sure. I'm always tempted to use an enum, but all it does is enable you to do `Enum.number` i.o. typing "number", which reduces the typos (because the IDE recognizes and auto suggests enum values), but other than that I can't think of anything